### PR TITLE
regen proto files to be in sync with grpc-go change 

### PR DESCRIPTION
**Pull request by maditya**
grpc-go commit that caused the incompatibility - https://github.com/grpc/grpc-go/commit/d84ff12005281b3411e2aafc893cc9cf28701a16

### CODE REVIEW

**Comment by maditya** [comment]
@andres-erbsen @daniel-ziegler please review!

File: keyserver/replication/raftlog/proto/raftrpc.pb.go [hunk 1]
```diff
@@ -16,17 +16,14 @@ package proto
 import proto1 "github.com/gogo/protobuf/proto"
```
Comment (maditya) [inline keyserver/replication/raftlog/proto/raftrpc.pb.go]: I had to add this line manually. Any idea why?

**Comment by adon-at-work** [comment]
I got an email notification for this PR (every file seems to have received a lot of changes ( e.g., M proto/client.pb.go (7890) ), and the surprisingly large number of changes (+11,497 −11,473) took me here. :-P

It seems almost every line of code is rewritten and have broken the diff calculations. 

My two cents: @maditya, my little advice is that can you please check what have caused such an issue (curious if it's due to some invisible line breaks or whitespaces...)? That'd help everyone to track/review what's being actually changed thru this PR. thanks. :)

**Comment by maditya** [comment]
@adon-at-work all these are auto-generated files by protoc. At this point, we don't want to pin to a particular version of protoc (which is built off of https://github.com/google/protobuf), grpc go (https://github.com/grpc/grpc-go/) or gogo proto (https://github.com/andres-erbsen/protobuf), all of which determine what gets generated. Also, we should never edit these files by hand regardless of whether it affects the diff or not (hence my question https://github.com/yahoo/coname/pull/28#discussion_r41939428). If we keep regenerating these more often you won't see such large changes, but it might still include some changes unrelated to our proto file change.

Also, there's nothing much to review in this PR except https://github.com/yahoo/coname/pull/28#discussion_r41939428 given the fact that we've not changed how we generate these files and picking up latest upstream packages is not breaking anything in our code.

**Comment by adon-at-work** [comment]
@maditya, thanks for the explanation. I see.

**Comment by maditya** [comment]
merging this for now, feel free to comment after.

**Pull request merged**